core/src/test/scala/unit/kafka/server/DescribeQuorumRequestTest.scala

```diff
@@ -60,6 +60,7 @@ class DescribeQuorumRequestTest(cluster: ClusterInstance) {
       val response = connectAndReceive[DescribeQuorumResponse](request)
 
       assertEquals(Errors.NONE, Errors.forCode(response.data.errorCode))
+      assertEquals("", response.data.errorMessage)
```

Comment:
In `DescribeQuorumResponse.json`, we said `null if no error` for the error message, but here it gets empty string. Where do we do that conversion?

Reply:
`Readable#readString` returns empty string if length of string equals to 0.
`DescribeQuorumResponseData#read` writes empty string to `errorMessage` for other version of RPC that not supports new field.